tree/src/package/mod.rs

```diff
@@ -0,0 +1,80 @@
+mod error;
+pub use error::{PackageError, PackageErrorType};
+
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Package {
+    pub name: String,
+    //section: String,
+    version: String,
+    release: Option<usize>, // Revision, but in apt's dictionary
```

Comment:
`release` should not be an `Option`. According to the `autobuild3`, when unset, it's `0` by default.